client/my-sites/patterns/components/pattern-preview/style.scss

```diff
@@ -18,6 +28,10 @@
 		overflow: hidden;
 	}
 
+	&:target .pattern-preview__renderer {
+		animation: 0.8s linear 3.5s backwards pattern-preview-target;
+	}
+
```

Comment:
I'm not sure what's wrong with this animation but I didn't notice anything when loading the page:

![video](https://github.com/Automattic/wp-calypso/assets/594356/20d1a348-4ea2-4c88-980a-a892cb703c68)


Interestingly, there is a blue border that briefly appears on Calypso.live which I think is really helpful to highlight the pattern. Unfortunately, it doesn't stay (and for some reason, I cannot see it when starting the branch locally). The page also jiggles which makes the experience less pleasant.

Reply:
Finally, I would try to make the page a little bit more reactive on mouse hover by adding the following:

```css
&:hover .pattern-preview__renderer {
  box-shadow: none;
}
```

And modifying the block below to add `&:not(:hover)` in front of the selector:

```css
&:not(:hover) .pattern-preview__copy {
  background-color: var(--default-color);
  border-color: var(--default-color);

  &:enabled:hover,
  &:focus {
    background-color: var(--hover-color);
    border-color: var(--hover-color);
  }
}
```

Reply:
> I'm not sure what's wrong with this animation

I went back and made a few tweaks that should hopefully improve that buggy behavior.

Reply:
> I would try to make the page a little bit more reactive on mouse hover

Hmm, I don't entirely see what we're trying to achieve there. Would you mind explaining further?

Reply:
The idea was to remove the shadow around the pattern, and use the original background color for the `Copy` button - when hovering a pattern. That way it gives more focus to the area the user is hovering. That kind of effect is a nice-to-have though, feel free to dismiss it.

Reply:
Oh okay, and use something like a gray color for the `Copy` buttons of the patterns that aren't being hovered?

Reply:
I think I would keep them in blue, that makes those call-to-action clearer to spot.